src/App.js

```diff
@@ -34,37 +34,40 @@ export default function App() {
   ] = useState([])
 
   useEffect(() => {
-    const historicCarbonFootprint = getFromStorage(
-      APP_STORAGE_KEYS.footprintHistory
-    )
-
-    historicCarbonFootprint && setCarbonFootprint(historicCarbonFootprint)
-
-    const historicTotalCarbonFootprint = getFromStorage(
-      APP_STORAGE_KEYS.footprintTotal
-    )
-
-    const historyFootprintPerTransportationType = getFromStorage(
-      APP_STORAGE_KEYS.footprintPerTransportType
-    )
-
-    setTotalCarbonFootprint(historicTotalCarbonFootprint)
-    setFootprintPerTransportationType(
-      historyFootprintPerTransportationType || []
-    )
-  }, [])
+    if (user) {
```

Comment:
please move the updateState and save method to an own service

Reply:
I tried that previously but for some reason I did not manage to update the third state. My approach was to write helper functions to update each state and give that to the service. For 2 of these functions it worked but for the third it always threw an error "is not a function". Thus, I left it in the App.js to make it finally work. 
If you have any suggestions for a better aproach I'd be happy to know 😄

Reply:
hmm, maybe it is a option for code review later